pages/dr10/files.rst

```diff
@@ -631,8 +673,14 @@ Users interested in database access to the Tractor `catalogs`_ can use the `NOIR
 Sweep Catalogs (``south/sweep/*``)
 ==================================
 
-10.0/sweep-<brickmin>-<brickmax>.fits
--------------------------------------
+10.[0-1]/sweep-<brickmin>-<brickmax>.fits
+-----------------------------------------
+
+.. note::
```

Comment:
Similarly to the comment above, you could put this note only once immediately below the 'Sweep Catalogs' section header.

Reply:
Again, I appreciated this opinion and have made your suggested update.